src/components/NavBar/NavigationBar.tsx

```diff
@@ -1,18 +1,29 @@
 import React, {useState, useEffect} from 'react';
 import './NavigationBar.css';
 import cehatLogo from './CEHATLogo.png';
-import menuIcon from './menuIcon.png';
 import {useTranslation} from 'react-i18next';
+import {FaBars, FaGlobeAmericas} from 'react-icons/fa';
 
+/** Element for the navigation bar for use on all pages */
 function NavigationBar(): JSX.Element {
+  // State for whether to use globe or text, must be kept
+  // separate because mobile should always be text, even
+  // when nav bar is hidden
+  const [isMobile, setIsMobile] = useState(
+    window.matchMedia('(max-width: 700px)')?.matches ?? false
+  );
+
   // State of nav bar (always visible in large screen)
-  const [isNavVisible, setIsNavVisible] = useState(true);
+  const [isNavVisible, setIsNavVisible] = useState(!isMobile);
 
+  /** Adjust UI for switching between narrow/mobile and wide/desktop modes */
   function handleScreenChange(this: MediaQueryList): void {
     if (this.matches) {
```

Comment:
Add a comment about what this.matches means here

Reply:
I'm not quite sure what it does since it wasn't my code originally but I have added a comment explaining what I think it does